Seer-Core/aql-processor/user-data-in/.gitignore

```diff
@@ -1,4 +0,0 @@
-# Ignore everything in this directory
-*
-# Except this file
-!.gitignore
```

Comment:
Do you need to delete this file? It was added so users don't have to create this directory by themselves. We have removed the instructions to create this dir.